bin/hsw-cli

```diff
@@ -592,93 +592,76 @@ class CLI {
       case 'rescan':
         await this.rescan();
         break;
+      case 'wallets':
+        await this.getWallets();
+        break;
+      case 'mkwallet':
+        await this.createWallet();
+        break;
+      case 'resend':
+        await this.resend();
+        break;
+      case 'backup':
+        await this.backup();
+        break;
+      case 'rpc':
+        await this.rpc();
+        break;
+
       default:
         this.log('Unrecognized command.');
+        this.log('');
         this.log('Commands:');
-        this.log('  $ listen: Listen for events.');
-        this.log('  $ get: View wallet.');
-        this.log('  $ master: View wallet master key.');
-        this.log('  $ shared add [xpubkey]: Add key to wallet.');
-        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
-        this.log('  $ balance: Get wallet balance.');
-        this.log('  $ history: View TX history.');
-        this.log('  $ pending: View pending TXs.');
-        this.log('  $ coins: View wallet coins.');
-        this.log('  $ account list: List account names.');
+        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ account create [account-name]: Create account.');
         this.log('  $ account get [account-name]: Get account details.');
+        this.log('  $ account list: List account names.');
         this.log('  $ address: Derive new address.');
```

Comment:
`address` and `change` require an `account` parameter, wanna add that? (It is required deep in hsd/wallet.js ... probably should be checked in wallet/http.js but 🤷‍♂️ )

Reply:
I'm for using 'default' if it's not provided explicitly.